entrypoint.sh

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 set -eu
 
-# Checking for host as first argument
+# ------------------------------
+# Check for DB host argument
+# ------------------------------
 if [ -z "${1:-}" ]; then
-  echo "Usage: $0 <host> [command...]"
+  echo "Usage: $0 <db_host> [command...]"
```

Comment:
The usage message mentions <db_host> but does not handle additional arguments. Clarify and support passing database host and other arguments properly.
```
if [ -z "${1:-}" ]; then
  echo "Usage: $0 <db_host> [command...]"
  exit 1
fi
```